libs/books/feature/src/lib/reading-list/reading-list.component.ts

```diff
@@ -9,10 +11,53 @@ import { getReadingList, removeFromReadingList } from '@tmo/books/data-access';
 })
 export class ReadingListComponent {
   readingList$ = this.store.select(getReadingList);
+  book:Book;
+  readingItem:ReadingListItem;
+  readingItems:ReadingListItem[];
+  constructor(private readonly store: Store, private snackBar: MatSnackBar) {}
 
-  constructor(private readonly store: Store) {}
-
-  removeFromReadingList(item) {
+  removeFromReadingList(item : ReadingListItem) {
     this.store.dispatch(removeFromReadingList({ item }));
+    
+    this.book={
```

Comment:
Again, shouldn't need to store a transient property like this. Better to keep things as stateless as possible.